libs/utils/src/cast_vote_records.ts

```diff
@@ -124,6 +131,11 @@ export function getWriteInsFromCastVoteRecord(
                   : pageIndex === 0
                   ? 'front'
                   : 'back',
+              isUnmarked:
+                selectionPosition.OtherStatus ===
+                UNMARKED_WRITE_IN_SELECTION_POSITION_OTHER_STATUS
+                  ? true
+                  : undefined,
```

Comment:
Why `true` or `undefined` vs just a boolean?

Reply:
Cool, I'd lean towards a required boolean since we always know one way or the other. When I see an optional field I have to think about if there's a diff between false and undefined or not